spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/system/DiskSpaceHealthIndicatorProperties.java

```diff
@@ -28,36 +30,54 @@
  *
  * @author Andy Wilkinson
  * @author Stephane Nicoll
+ * @author Chris Bono
  * @since 1.2.0
  */
 @ConfigurationProperties(prefix = "management.health.diskspace")
 public class DiskSpaceHealthIndicatorProperties {
 
 	/**
-	 * Path used to compute the available disk space.
+	 * Paths to consider for computing the available disk space.
 	 */
-	private File path = new File(".");
+	private List<PathInfo> paths = Arrays.asList(new PathInfo());
```

Comment:
As stated in the overview comment, we could keep support of the `management.health.diskspace.path` (for singular case only) by mapping a `private PathInfo path = new PathInfo();` here as well.